server/config.js

```diff
@@ -3,7 +3,73 @@ const { tmpdir } = require('os');
 const path = require('path');
 const { randomBytes } = require('crypto');
 
+const passwordConvict = convict({
+  maxLength: {
+    type: {
+      format: ['maxLength']
+    },
+    length: {
+      doc: 'the maximum length allowed for passwords',
+      format: Number,
+      default: 32
+    }
+  },
+  minLength: {
+    type: {
+      format: ['minLength']
+    },
+    length: {
+      doc: 'the maximum length allowed for passwords',
```

Comment:
noticed this documentation is the same as for maxLength

Reply:
fixed in master